ci-operator/config/backube/scribe/backube-scribe-master.yaml

```diff
@@ -1,37 +1,104 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.8"
+    namespace: ocp
+    tag: cli
+  ocp_builder_rhel-8-golang-1.16-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.8
+  os:
+    name: centos
+    namespace: origin
+    tag: "8"
+  ubi_minimal:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
 binary_build_commands: make all
 build_root:
   project_image:
     dockerfile_literal: |
       FROM registry.fedoraproject.org/fedora:33
       USER root
       RUN dnf install -y ShellCheck yamllint asciidoctor && \
-          dnf install -y make go diffutils && \
+          dnf install -y make go diffutils openssl && \
           dnf clean all && rm -rf /var/cache/dnf/*
+      RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && \
+          chmod 700 get_helm.sh && \
+          ./get_helm.sh
       ENV GOPATH=/go GOCACHE=/go/src/.cache GO111MODULE=on
       RUN mkdir -p $GOCACHE /.cache $GOPATH/src/github.com/backube/testbin
       RUN chmod -R 777 /go /.cache
       RUN curl -sSLo envtest-bins.tar.gz "https://storage.googleapis.com/kubebuilder-tools/kubebuilder-tools-1.19.2-$(go env GOOS)-$(go env GOARCH).tar.gz"
       RUN tar -C $GOPATH/src/github.com/backube/testbin --strip-components=1 -zvxf envtest-bins.tar.gz
       WORKDIR $GOPATH/src/github.com/backube/scribe
```

Comment:
^^ leaving the steps that configure the image to run the unit tests, in case we want to run unit tests here in the future